tests/python/frontend/tflite/test_forward.py

```diff
@@ -4317,10 +4326,10 @@ def convert_sub_dummy(self, op):
         ]
         out = math_ops.subtract(in_data[0], in_data[1])
         in_name = [x[1] for x in zip(in_data, ("in_0:0", "in_1:0"))]
-        input_tensors = [x for x in in_data]
+        input_tensors = in_data
         output_tensors = [out]
         in_node = [0] * len(in_name)
-        for i in range(len(in_name)):
+        for i, _ in enumerate(in_name):
             in_node[i] = in_name[i].split(":")[0] if ":" in in_name[i] else in_name[i]
```

Comment:
```suggestion
            in_node[i] = in_name[i].split(":")[0]
```
If ":" is not found, split will return [in_node_[i]]